extensions/wikia/Search/classes/Test/ResultTest.php

```diff
@@ -465,15 +458,11 @@ public function testGetVideoViewsException() {
 	/**
 	 * @group Slow
 	 * @slowExecutionTime 0.09983 ms
-	 * @covers Wikia\Search\Result::offsetGet
+	 * @covers \Wikia\Search\Result::offsetGet
 	 */
 	public function testOffsetGetTitle() {
 		$result = $this->getMock( 'Wikia\\Search\\Result', [ 'getTitle' ] );
-		$result
-			->expects( $this->any() )
-			->method ( 'getTitle' )
-			->will   ( $this->returnValue( 'my title' ) )
-		;
+		$result->expects( $this->any() )->method( 'getTitle' )->will( $this->returnValue( 'my title' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "my title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)